### PR TITLE
Adding Assignment 7 And 8

### PR DESCRIPTION
**Pull request by somesh6797**

### CODE REVIEW

File: Somesh/HTML/Assignment7/indexstyle.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    align-items: center;
+    width: 70vw;
+    border-radius: 20px;
+    background: rgba(255, 255, 255, 0.25);
+    box-shadow: 0 8px 32px 0 rgba(0, 0, 0, 0.37);
+    backdrop-filter: blur(4px);
+    border-radius: 10px;
+    border: 1px solid rgba(255, 255, 255, 0.18);
+}
+
```
Comment (kalyani1942) [inline Somesh/HTML/Assignment7/indexstyle.css:70]: Remove Space

File: Somesh/HTML/Assignment7/login.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
+body {
+    background-color: rgb(0, 175, 123);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 97vh;
+}
+
```
Comment (kalyani1942) [inline Somesh/HTML/Assignment7/login.css]: Remove Space

File: Somesh/HTML/Assignment7/login.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                placeholder="Password"
+            /><br />
+            <button class="btn"">Login</button>
+            <div class="text">
+                <a href="#">Forgot Password?</a><br />
+                <a href="#">Don't have an account? &nbsp;Create One</a>
+            </div>
+        </div>
+    </body>
+
```
Comment (kalyani1942) [inline Somesh/HTML/Assignment7/login.html]: Remove Space

File: Somesh/HTML/Assignment7/personal-details.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                    <!-- gender:- -->
+                    <input type="radio" id="male" />
+                    <label for="male">Male </label>
+                    <input type="radio" id="Female" />
+                    <label for="Female">Female </label>
+                    <input type="radio" id="other" />
+                    <label for="other">Prefer not to choose </label>
+                </div>
+                <input type="text" placeholder="Age" />
+
```
Comment (kalyani1942) [inline Somesh/HTML/Assignment7/personal-details.html:27]: remove Extra Line

File: Somesh/HTML/Assignment7/style.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    background-image: linear-gradient(
+            rgba(82, 86, 173, 0.3),
+            rgba(82, 86, 173, 0.3)
+        ),
+        url("meet.jpg");
+    background-size: cover;
+    background-position: center;
+    height: 95vh;
+}
+
```
Comment (kalyani1942) [inline Somesh/HTML/Assignment7/style.css]: So much Extra Line

**Review by kalyani1942 — CHANGES_REQUESTED** [review]

**Review by kalyani1942 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**